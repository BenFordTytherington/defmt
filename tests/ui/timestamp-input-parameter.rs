#![no_main]
#[defmt::timestamp]
fn foo(x: u64) -> u64 { x + 1 }
