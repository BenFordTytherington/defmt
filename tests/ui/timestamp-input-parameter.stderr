error: function must have signature `fn() -> u64`
 --> $DIR/timestamp-input-parameter.rs:3:4
  |
3 | fn foo(x: u64) -> u64 { x + 1 }
  |    ^^^
